src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

```diff
@@ -63,13 +72,12 @@ source:
     location: http://host.net/path/project-src.zip
 ----
 
-=== Other project parameters
 
-Also each project may have `clonePath` parameter, which specifies the path into which the project should be cloned.
-It should be relative to `/projects` directory and cannot leave it.
-By default it is equal to project name.
+=== Project clone-path parameter: clonePath
+
+The `clonePath` parameter specifies the path into which the project is to be cloned. The path must be relative to the `/projects` directory, and it cannot leave the `projects/` directory. The default value is the project name.
```

Comment:
Maybe it is clearer to have `/projects/` instead of `projects/`

Reply:
Fixed.